configs/defaults/large_cohort.toml

```diff
@@ -53,6 +53,12 @@ pca_plot_name = 'hgdp_1kg_sites'
 # set whether PCA output labels are from given population names, or random forest
 # inferred population labels
 inferred_ancestry = false
+# Whether or not to infer ploidy using the variant data:
+# Using variant sites only for ploidy calculations causes ChrY ploidy inflation.
+# Setting `variants_only_x_ploidy` and `variants_only_y_ploidy` to `False` (default value) causes
+# function to use reference blocks for ploidy calculations
+# https://centrepopgen.slack.com/archives/C018KFBCR1C/p1705539231990829?thread_ts=1704233101.883849&cid=C018KFBCR1C
+inf_ploidy_using_var = true
```

Comment:
shouldn't this be False? ie we `variant_only...` to be set to False.

Reply:
Correct